package.json

```diff
@@ -5,21 +5,31 @@
   "type": "commonjs",
   "source": "src/index.tsx",
   "main": "dist/index.js",
-  "module": "dist/esm/index.js",
+  "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
   "exports": {
```

Comment:
this entry does not work for "moduleResolution: node", that's why it's important to make "test-utils" a directory with "index.js" file inside it. All that is configured inside `tsup` config

Reply:
This "hack" was done to fix another issue, with Webpack not understanding `.mjs` https://github.com/thebuilder/react-intersection-observer/issues/633

Reply:
Modern tooling would use the `exports` field over the `module`, which is why it's sorta stuck in a limbo legacy hacks for ESM.

Reply:
Ok. I can make a copy: "dist/esm/index.js" for that case, I think

Reply:
`module` value restored in [a66dcae](https://github.com/thebuilder/react-intersection-observer/pull/695/commits/a66dcae3d347fd0be2d9f8129588e77e33f6a3fc), @thebuilder